### DISCUSSION TITLE
How can we integrate the Argocd with AWS Secret Manager

### DISCUSSION THREAD

**Discussion by shwethaganiga** [argoproj/argo-cd#15477] [Q&A]
Hi All,
Can we integrate the Argocd with AWS Secret Manager. Please help me if anybody have prior experience on this. Thanks

**Comment by shwethaganiga** [comment]
Hi @all,
Please help me o this.

**Reply by bainss** [reply]
use external secrets operator https://external-secrets.io/latest/ for this ...https://github.com/polarpoint-io/argocd-app-of-apps/tree/main/templates

**Comment by shwethaganiga** [comment]
okay Thanks @bainss
